example/App.tsx

```diff
@@ -86,6 +86,7 @@ import { Icon } from '@swmansion/icons';
 interface Example {
   name: string;
   component: React.ComponentType;
+  unsupportedPlatforms?: Set<String>;
```

Comment:
```suggestion
  unsupportedPlatforms?: Set<typeof Platform.OS>;
```

Reply:
Done in [f475fb0](https://github.com/software-mansion/react-native-gesture-handler/pull/3135/commits/f475fb0ad88a2515494b97bb23713a2ee6f7c341)